_rules/SC4-1-1-unique-attrs.md

```diff
@@ -0,0 +1,135 @@
+---
+name: attributes are not duplicated
+rule_type: atomic
+description: |
+  This rule checks that HTML and SVG starting tags do not contain duplicated attributes.
+
+success_criterion:
+- 4.1.1 # Parsing
+
+test_aspects:
+- Source code
+
+authors:
+- Wilco Fiers
+- Emma Pratt Richens
+---
+
+
+## Applicability
+
+The rule applies to any [starting tag](https://www.w3.org/TR/html5/syntax.html#start-tags) in an HTML or SVG file.
+
+**Note:** This rule cannot be tested on the DOM Tree because the browser ignores duplicated attributes.
```

Comment:
Per the HTML syntax specification, if an attribute name that is already present on a start/end tag is encountered, the attribute is dropped (http://w3c.github.io/html/syntax.html#attribute-name-state). As such, only the first encountered attribute is used if multiple attributes with the same name are defined. That's not quite the same as _ignoring_ duplicated attributes if one regards the term "duplicate" as encompassing the entire set of things that are equal.